cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/sponge/platform/SpongeClientServerPingEventCaller.java

```diff
@@ -0,0 +1,79 @@
+package de.dytanic.cloudnet.ext.bridge.sponge.platform;
+
+import de.dytanic.cloudnet.ext.bridge.BridgeHelper;
+import de.dytanic.cloudnet.ext.bridge.server.BridgeServerHelper;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.event.SpongeEventFactory;
+import org.spongepowered.api.event.cause.Cause;
+import org.spongepowered.api.event.cause.EventContext;
+import org.spongepowered.api.event.server.ClientPingServerEvent;
+import org.spongepowered.api.network.status.StatusResponse;
+import org.spongepowered.api.scheduler.SpongeExecutorService;
+
+import java.util.concurrent.TimeUnit;
+
+public final class SpongeClientServerPingEventCaller {
```

Comment:
Why does only Sponge have a seperate class for this?
Furthermore, this should be done with an Object which receives the required information (like the plugin instance) via the constructor.
In my opinion, this is a better software design.

Reply:
Sponge hat eine separate Klasse da auf package-private Methoden für die Interne API zugegriffen wird.

Reply:
I also use static methods here, because we never use multiple instances of this class and we can also better control the external calling of the start method.

Reply:
it doesn't matter if we actually never create a second instance, it's just prettier software design.

Reply:
It is simply unnecessary at this point